src/mono/wasm/runtime/exports.ts

```diff
@@ -0,0 +1,238 @@
+import { mono_wasm_add_event_listener, mono_wasm_remove_event_listener } from './event-listener'
+import { mono_wasm_release_cs_owned_object } from './gc-handles'
+import { mono_bind_method, _create_primitive_converters } from './method-binding'
+import { mono_wasm_web_socket_open, mono_wasm_web_socket_send, mono_wasm_web_socket_receive, mono_wasm_web_socket_close, mono_wasm_web_socket_abort } from './web-socket'
+import cwraps from './cwraps'
+import { runtimeHelpers } from './corebindings'
+
+export let MONO: MONO = <any>{
+    // current "public" MONO API
+    mono_wasm_setenv,
+    mono_wasm_load_bytes_into_heap,
+    mono_wasm_load_icu_data,
+    mono_wasm_runtime_ready,
+    mono_wasm_load_data_archive,
+    get loaded_files() {
+        return runtimeHelpers.loaded_files
+    },
+
+    // EM_JS macro
+    string_decoder,
+
+    // generated bindings closure `library_mono`
+    mono_wasm_new_root_buffer,
+    mono_wasm_new_root_buffer_from_pointer,
+    mono_wasm_new_root,
+    mono_wasm_new_roots,
+    mono_wasm_release_roots,
+    //mono.mono_wasm_get_icudt_name = mono_wasm_get_icudt_name,
+
+    // used in debugger
+    mono_wasm_get_loaded_files,
+    mono_wasm_add_dbg_command_received,
+    mono_wasm_send_dbg_command_with_parms,
+    mono_wasm_send_dbg_command,
+    mono_wasm_get_dbg_command_info,
+    mono_wasm_get_details,
+    mono_wasm_release_object,
+    mono_wasm_call_function_on,
+    mono_wasm_debugger_resume,
+    mono_wasm_detach_debugger,
+    mono_wasm_raise_debug_event,
+    get mono_wasm_runtime_is_ready() {
+        return runtimeHelpers.mono_wasm_runtime_is_ready
+    },
+
+    // used in tests
+    mono_load_runtime_and_bcl_args,
+    mono_wasm_load_config,
+    mono_wasm_set_runtime_options,
+    get config() {
+        return runtimeHelpers.config
+    },
+}
+
+export let BINDING: BINDING = <any>{
+    //current "public" BINDING API
+    mono_obj_array_new: cwraps.mono_wasm_obj_array_new,
+    mono_obj_array_set: cwraps.mono_wasm_obj_array_set,
+    js_string_to_mono_string,
+    js_typed_array_to_array,
+    js_to_mono_obj,
+    mono_array_to_js_array,
+    conv_string,
+    bind_static_method: mono_bind_static_method,
+    call_assembly_entry_point: mono_call_assembly_entry_point,
+    unbox_mono_obj,
+
+    // generated bindings closure `binding_support`
+    _get_args_root_buffer_for_method_call,
+    _get_buffer_for_method_call,
+    invoke_method: cwraps.mono_wasm_invoke_method,
+    _handle_exception_for_call,
+    mono_wasm_try_unbox_primitive_and_get_type: cwraps.mono_wasm_try_unbox_primitive_and_get_type,
+    _unbox_mono_obj_root_with_known_nonprimitive_type,
+    _teardown_after_call,
+
+    // tests
+    call_static_method,
+    mono_intern_string,
+
+    // startup
+    BINDING_ASM: "[System.Private.Runtime.InteropServices.JavaScript]System.Runtime.InteropServices.JavaScript.Runtime",
+}
+
+export function export_to_emscripten(mono: any, binding: any, dotnet: any, module: t_ModuleExtension) {
```

Comment:
Please add a comment that explains what this function does, I can't figure it out